src/stirling/protocol_inference/data_collection.py

```diff
@@ -87,5 +100,5 @@ def trace_pods(pod2pid, node, pods, duration):
     node = args.node
     duration = args.duration
 
-    pod2pid = create_pod2pid()
+    pod2pid = create_pod2pid(args.container_runtime)
     trace_pods(pod2pid, node, pods, duration)
```

Comment:
Could convert to using `fire` ... optional, but, it is a very concise way to handle cmd line args.